extensions/fairifier-rdf-extension/dtls/fairifier/FileSystemRdfSkeletonImpl.java

```diff
@@ -1,95 +1,79 @@
+
 package org.dtls.fairifier;
 
-import java.io.BufferedWriter;
 import java.io.File;
-import java.io.FileWriter;
 import java.io.IOException;
 import java.io.UncheckedIOException;
-import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import java.util.Optional;
 
 public class FileSystemRdfSkeletonImpl implements RdfSkeletonService {
+
     private static final Path SAVELOCATION = Paths.get(System.getProperty("user.home") + "/.local/share/openrefine/");
     private static final ObjectMapper mapper = new ObjectMapper();
-    private static final RdfSkeletonTransformer rdfSkeletonJsonTransformerImpl = new  RdfSkeletonJsonTransformerImpl(); 
-    
+    private static final RdfSkeletonTransformer rdfSkeletonJsonTransformerImpl = new RdfSkeletonJsonTransformerImpl();
```

Comment:
With the current setup this approach is not thread-safe. See the comment on the `RdfSkeletonJsonTransformerImpl ` class for how to solve this.